src/main/frontend/modules/outliner/core.cljs

```diff
@@ -351,44 +351,34 @@
       (zip/left result-loc-or-vec)
       result-loc-or-vec)))
 
-(defn- get-node-tree-sub-topmost-last-loc
```

Comment:
clj-kondo catches unused private vars. There are a number of them I've been seeing. At some point, we could have a separate CI step that catches unused public vars like https://github.com/cldwalker/bb-clis/blob/master/bin/bb-unused-vars

Reply:
That'll be great!